lib/OpenLoans/OpenLoans.js

```diff
@@ -204,14 +204,16 @@ class OpenLoans extends React.Component {
     }
 
     return (
-      <Row>
-        <Col xs={2}>
-          <Label>{this.props.loans.length} Records found</Label>
-        </Col>
-        <Col xs={10}>
-          <Button id="renew-all" disabled={isDisabled} title={stringToolTip} onClick={this.renewAll}>Renew</Button>
-        </Col>
-      </Row>
+      <div style={{ padding: '15px 15px 0' }}>
```

Comment:
Should move this out to a css file at some point, but doesn't block merging this.

Reply:
You are right. That's just temporarily. I think some design changes are coming to this part anyway.